sdk/search/azure-search-documents/README.md

```diff
@@ -325,6 +325,26 @@ print("Upload of new document succeeded: {}".format(result[0].succeeded))
 ```
 
 
+### Async APIs
+This library includes a complete async API supported on Python 3.5+. To use it, you must
+first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/).
+See
+[azure-core documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md#transport)
+for more information.
+
+
+```py
+from azure.search.documents.aio import SearchClient
+
+client = SearchClient(
+            endpoint, index_name, AzureKeyCredential(api_key)
+        )
+
+async with client:
+  results = await client.search(search_text="hotel")
```

Comment:
If "results" is an async iterator, would benefit one step further with "async for"